assets/src/scss/layout/_footer.scss

```diff
@@ -147,7 +147,7 @@
   --footer--copyright-- {
     color: $grey-10;
   }
-  font: $font-size-xxs;
+  font-size: $font-size-xxs;
```

Comment:
The `font-size` property has not been set correctly since the beginning of this implementation.

![footer-error](https://user-images.githubusercontent.com/77975803/133485065-80ef87a1-ecf9-45b9-b64b-1cb32c9fd681.png)

Reply:
Good catch :ok_hand: I suppose we can do this change as it looks better. Maybe leave a comment on the ticket :pray: